design/httpproxy-conflict.md

```diff
@@ -0,0 +1,110 @@
+# HTTPProxy Conflict Resolution Design
+
+Status: Draft
+
+## Abstract
+
+Configuration errors in HTTPProxy resources can result in objects being marked as invalid and consequently will not serve traffic.
+These errors can come from invalid route configuration, missing service references, or invalid includes to other resources. 
+
+## Background
+HTTPProxy is designed as a multi-team aware Ingress that reduces annotation sprawl by providing a sensible home for configuration items.
+Additionally, HTTPProxy offers a feedback loop to users via a `Status` field on the object.
```

Comment:
nit: related to my other comment about personas, here `users` is a bit overloaded?

Reply:
Nit: Going by the new Personas section, this should be updated to ResourceOwner? Or, if you don't want to do that until you've defined it, maybe "HTTPProxy users" or HTTPProxy Owners would be more correct.